src/containers/app.jsx

```diff
@@ -185,13 +185,11 @@ class App extends Component {
     }
 
     return (
-      <>
-        <MobileScreenFallback>
-          <IntlProvider locale="de" messages={localeDe}>
-            <div className={styles.app}>{this.renderView()}</div>
-          </IntlProvider>
-        </MobileScreenFallback>
-      </>
+      <MobileScreenFallback>
+        <IntlProvider locale="de" messages={localeDe}>
+          <div className={styles.app}>{this.renderView()}</div>
+        </IntlProvider>
+      </MobileScreenFallback>
```

Comment:
Oh, my bad. Thanks for catching this! In the earlier version, the `MobileScreen` was next to the `IntlProvider` and the fragment was required.

Reply:
I found an eslint rule for that: [react/jsx-no-useless-fragment](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-useless-fragment.md) 🎉 Should I enable it?

Reply:
Yes, sounds like a good rule!